docs/VALUES.md

```diff
@@ -0,0 +1,32 @@
+# Core Values
+Shared by a broader community of the Organic Maps project developers and users.
+
+- [Free and Open Source](#free-and-open-source)
+- [Privacy](#privacy)
+- [Community](#community)
+- Core qualities of user-facing Organic Maps apps:
+  - Easy to use
+  - Offline-centric
+  - Fast and light on resources
+  - Privacy-respecting
+
+# Free and Open Source
+The complete Organic Maps project codebase and all associated tools, documents and resources are released as free-as-in-freedom software using Free Software licenses.
+All data comes from open sources (e.g. map data is based on OpenStreetMap).
+
+*At the moment non-F-Droid Android builds of OM contain a proprietary Google Play Services Location library, which possible could be replaced by a FOSS implementation in the future.
+Please let us know if you spot any other non-free bits.
+
+**The data file associating OSM hotel POIs with Kayak listings is courtesy of Kayak.com.
+
+# Privacy
+All personal information is protected by default to a reasonable extent.
+There are additional options for sensitive users to further enhance protection by sacrificing some convenience / ease of use.
+No personal information is collected without user's explicit consent.
+Organic Maps might collect anonymous usage statistics in a privacy-preserving way. These stats couldn't possibly be used to identify or track a user.
+
+The project pledges to disclose all potential limitations on data protection and cases of anonymous data collection.
```

Comment:
I'm planning to link to a detailed privacy disclosure document from here.
Its WIP.

Reply:
> I would start with the actual legally right privacy policy first.

Sure! But its likely to be two related but different documents: a legal privacy policy and a detailed technical disclosure.

Reply:
> Hey, here is an entire section from the original project charter

Thanks for sharing this!
I'll think how to best integrate points from there into the Core Values and other docs!